litmus-portal/frontend/src/pages/MonitoringDashboardPage/refreshData.ts

```diff
@@ -0,0 +1,42 @@
+export default [
+  {
+    label: 'Every 5 seconds',
+    value: 5000,
+  },
+  {
+    label: 'Every 10 seconds',
+    value: 10000,
+  },
+  {
+    label: 'Every 30 seconds',
+    value: 30000,
+  },
+  {
+    label: 'Every 1 minute',
+    value: 60000,
+  },
+  {
+    label: 'Every 5 minutes',
+    value: 300000,
+  },
+  {
+    label: 'Every 15 minutes',
+    value: 900000,
+  },
+  {
+    label: 'Every 30 minutes',
+    value: 1800000,
+  },
+  {
+    label: 'Every 1 hour',
+    value: 3600000,
+  },
+  {
+    label: 'Every 2 hours',
+    value: 7200000,
+  },
+  {
+    label: 'Every 1 day',
+    value: 86400000,
+  },
```

Comment:
all the labels should come from translation

Reply:
here you can't use translation as this is not a react component

Reply:
This particular component should be constructed separately.

Reply:
it's just an object...component not needed @whitetiger1399

Reply:
yes i will add the translations.

Reply:
Added.